src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -20,99 +20,534 @@ PICS:
+                        EpochStartTime1: 0,
+                        EpochKey2: "\xc0\xc1\xc2\xc3\xc4\xc5\xc6\xc7\xc8\xc9\xca\xcb\xcc\xcd\xce\xcf",
+                        EpochStartTime2: 0,
+                    }
 
-      disabled: true
+    - label:
+          "Preparation step for using commands from Groups cluster: Write Group
+          Keys"
+      cluster: "Group Key Management"
+      endpoint: 0
+      command: "writeAttribute"
+      attribute: "GroupKeyMap"
+      arguments:
+          value:
+              [
+                  { FabricIndex: 1, GroupId: 0x0001, GroupKeySetID: 0x01a1 },
+                  { FabricIndex: 1, GroupId: 0x0002, GroupKeySetID: 0x01a1 },
+              ]
 
     - label:
-          "TH sends a AddGroup command to DUT with the GroupID field set to G1."
+          "TH sends a AddGroup command to DUT with the GroupID field set to
+          0x0001."
       PICS: G.S.C00.Rsp
-      verification: |
-          DUT sends a AddGroupResponse command to TH with the Status field set to 0x00 (SUCCESS) or 0x8a (DUPLICATE_EXISTS) and the GroupID field set to G1.
-      disabled: true
+      cluster: "Groups"
+      command: "AddGroup"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "GroupName"
+                value: "Group1"
+      response:
+          values:
+              - name: "Status"
+                value: 0
+              - name: "GroupID"
+                value: 0x0001
 
     - label:
           "TH sends a RemoveAllScenes command to DUT with the GroupID field set
-          to G1."
+          to 0x0001."
       PICS: S.S.C03.Rsp
-      verification: |
-          DUT sends a RemoveAllScenesResponse command to TH with the Status field set to 0x00 (SUCCESS) and GroupID field set to G1.
-      disabled: true
+      command: "RemoveAllScenes"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+      response:
+          values:
+              - name: "Status"
+                value: 0
+              - name: "GroupID"
+                value: 0x0001
 
     - label:
           "TH sends a GetSceneMembership command to DUT with the GroupID field
-          set to G1."
+          set to 0x0001."
       PICS: S.S.C06.Rsp
-      verification: |
-          DUT sends a GetSceneMembershipResponse command to TH with the Status field set to 0x00 (SUCCESS), the Capacity field set to the value SC0, the GroupID field set to G1 and the SceneCount field set to 0x00.
-      disabled: true
+      command: "GetSceneMembership"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "Capacity"
+                value: (maxScenes/2)
+              - name: "GroupID"
+                value: 0x0001
```

Comment:
Please file an issue on the test plan: it has:

> DUT sends a GetSceneMembershipResponse  ... the SceneCount field set to 0x00

which does not match the spec.